SKILL.tsv

```diff
@@ -1089,35 +1089,35 @@ SKILL_20150317_001088	Fire Ball
 SKILL_20150317_001089	{#993399}{ol}[Magic] - [Fire]{/}{/}{nl} Summon Fire sphere. You can Blow the sphere using sword attack.
 SKILL_20150317_001090	Attack #{SkillAtkAdd}#{nl}Attack #{SkillSR}# times{nl}Duration #{CaptionTime}# seconds
 SKILL_20150317_001091	Fire Wall
-SKILL_20150317_001092	{#993399}{ol}[Magic] - [Fire]{/}{/}{nl} Create flame barrier to damage adjacent enemies.
+SKILL_20150317_001092	{#993399}{ol}[Magic] - [Fire]{/}{/}{nl} Creates a flame barrier inflicting damage to a nearby enemy.
 SKILL_20150317_001093	Attack #{SkillAtkAdd}#{nl}#{CaptionRatio}# Fire Wall #{nl}Attack #{CaptionRatio2}# times{nl}Duration 15 seconds
 SKILL_20150317_001094	Enchant Fire
-SKILL_20150317_001095	Grant Fire Property attack force to target. Additional damage is applied for basic attack.
+SKILL_20150317_001095	Grants the Fire attack attribute to the target. Additional damage applies to your basic attack.
 SKILL_20150317_001096	Fire Property attack + #{CaptionRatio}#{nl} Additional Damage + #{CaptionRatio2}#{nl}Duration #{CaptionTime}# seconds
 SKILL_20150317_001097	Flare
-SKILL_20150317_001098	{#993399}{ol}[Magic] - [Fire]{/}{/}{nl} Explode enemy in abnornal state except petrified and frozen state.
+SKILL_20150317_001098	{#993399}{ol}[Magic] - [Fire]{/}{/}{nl} Inflict damage by exploding the enemy affected by a status effect. Excluding both petrified and frozen status effects.
```

Comment:
Inflicts? Also, extremely long and confusing. Possibly replace with "Inflicts damage by exploding enemies afflicted by  "Burning" status"?